Resources.it.resx

```diff
@@ -0,0 +1,1044 @@
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AppName" xml:space="preserve">
+    <value>Sidebar Diagnostics</value>
+    <comment>Name of the app</comment>
+  </data>
+  <data name="ButtonApply" xml:space="preserve">
+    <value>Applica</value>
+    <comment>Text of the apply button</comment>
+  </data>
+  <data name="ButtonCancel" xml:space="preserve">
+    <value>Annulla</value>
+    <comment>Text of the cancel button</comment>
+  </data>
+  <data name="ButtonClose" xml:space="preserve">
+    <value>Chiudi</value>
+    <comment>Text of the close button</comment>
+  </data>
+  <data name="ButtonDonate" xml:space="preserve">
+    <value>Donare</value>
+    <comment>Text of the donate button</comment>
+  </data>
+  <data name="ButtonDone" xml:space="preserve">
+    <value>Fatto</value>
+    <comment>Text of the done button</comment>
+  </data>
+  <data name="ButtonGitHub" xml:space="preserve">
+    <value>GitHub</value>
+    <comment>Text of the GitHub button</comment>
+  </data>
+  <data name="ButtonGraph" xml:space="preserve">
+    <value>Grafico</value>
+    <comment>Text of the graph button</comment>
+  </data>
+  <data name="ButtonHide" xml:space="preserve">
+    <value>Nascondi</value>
+    <comment>Text of the hide button</comment>
+  </data>
+  <data name="ButtonNo" xml:space="preserve">
+    <value>No</value>
+    <comment>Text of the no button</comment>
+  </data>
+  <data name="ButtonReload" xml:space="preserve">
+    <value>Ricarica</value>
+    <comment>Text of the reload button</comment>
+  </data>
+  <data name="ButtonSave" xml:space="preserve">
+    <value>Salva</value>
+    <comment>Text of the save button</comment>
+  </data>
+  <data name="ButtonSettings" xml:space="preserve">
+    <value>Impostazioni</value>
+    <comment>Text of the settings button</comment>
+  </data>
+  <data name="ButtonShow" xml:space="preserve">
+    <value>Mostra</value>
+    <comment>Text of the show button</comment>
+  </data>
+  <data name="ButtonSkip" xml:space="preserve">
+    <value>Salta</value>
+    <comment>Text of the skip button</comment>
+  </data>
+  <data name="ButtonUpdate" xml:space="preserve">
+    <value>Aggiorna</value>
+    <comment>Text of the update button</comment>
+  </data>
+  <data name="ButtonVisibility" xml:space="preserve">
+    <value>Visibilità</value>
+    <comment>Text of the visibility button</comment>
+  </data>
+  <data name="ButtonYes" xml:space="preserve">
+    <value>Sì</value>
+    <comment>Text of the yes button</comment>
+  </data>
+  <data name="ChangeLogTitle" xml:space="preserve">
+    <value>Change Log</value>

```

Comment:
'Cambia Registro'

Reply:
'Change Log' is a specific term that I would not translate.

Reply:
How about 'Modifica il Log'?

Reply:
Nono, Change Log is uesd in italian.